code/__HELPERS/text.dm

```diff
@@ -818,12 +833,12 @@ GLOBAL_LIST_INIT(binary, list("0","1"))
 	catch
 		return
 
-/proc/num2loadingbar(percent as num, var/numSquares = 20, var/reverse = FALSE) 
-	var/loadstring = "" 
-	for (var/i in 1 to numSquares) 
+/proc/num2loadingbar(percent as num, var/numSquares = 20, var/reverse = FALSE)
+	var/loadstring = ""
+	for (var/i in 1 to numSquares)
 		var/limit = reverse ? numSquares - percent*numSquares : percent*numSquares
-		if (i <= limit) 
-			loadstring += "█" 
-		else 
-			loadstring += "░" 
-	return "\[" + loadstring + "]" 
+		if (i <= limit)
+			loadstring += "█"
+		else
+			loadstring += "░"
+	return "\[" + loadstring + "]"

```

Comment:
```suggestion
		loadstring += i <= limit ? "█" : "░"
	return "\[[loadstring]\]"
```

Reply:
This only got touched due to LF but alright.